asserts/ifacedecls.go

```diff
@@ -434,10 +441,10 @@ func compilePlugRule(interfaceName string, rule interface{}) (*PlugRule, error)
 		return nil, err
 	}
 	if rMap == nil {
-		if shortcut {
-			rMap = defaultAllowRule
-		} else {
-			rMap = denyRule
+		if shortcut == "true" {
+			rMap = defaultOutcome
+		} else { // "false"
```

Comment:
Why the change out of bool?  It looks like we're using it as a bool still, even assuming the alternative value when it's not "true", so it's not clear what we won here?